apps/web/src/common/modules/navigations/lnb/LNB.vue

```diff
@@ -33,7 +33,9 @@ const props = withDefaults(defineProps<Props>(), {
     menuSet: () => [],
     showFavoriteOnly: undefined,
 });
-const emit = defineEmits<{(e: 'select', id: string, selected: string|number): void}>();
+const emit = defineEmits<{(e: 'select', id: string, selected: string|number): void;
+    (e: 'update:showFavoriteOnly', value: boolean): void;
```

Comment:
suggestion: We have a convention to use kebab case for the event name. 
Is there an issue if we use kebab case here? If there is, ignore it.

Reply:
Oh.. thanks..

Reply:
My mistake